ios/Exponent/Supporting/kernel-manifest.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"android":{"package":"host.exp.exponent","publishBundlePath":"../android/app/src/main/assets/kernel.android.bundle"},"description":"","extra":{"amplitudeApiKey":"081e5ec53f869b440b225d5e40ec73f9"},"icon":"https://s3.amazonaws.com/exp-brand-assets/ExponentEmptyManifest_192.png","iconUrl":"https://s3.amazonaws.com/exp-brand-assets/ExponentEmptyManifest_192.png","ios":{"bundleIdentifier":"dev.expo.Payments","publishBundlePath":"../ios/Exponent/Supporting/kernel.ios.bundle","supportsTablet":true},"isKernel":true,"kernel":{"androidManifestPath":"../android/app/src/main/assets/kernel-manifest.json","iosManifestPath":"../ios/Exponent/Supporting/kernel-manifest.json"},"locales":{},"name":"expo-home","orientation":"portrait","packagerOpts":{"config":"rn-cli.config.js"},"platforms":["ios","android"],"primaryColor":"#cccccc","privacy":"unlisted","scheme":"exp","sdkVersion":"UNVERSIONED","slug":"home","updates":{"checkAutomatically":"ON_LOAD","fallbackToCacheTimeout":0},"version":"32.0.0","id":"@exponent/home","revisionId":"32.0.0-r.ygzPNBGrYV","publishedTime":"2019-01-12T16:19:01.738Z","commitTime":"2019-01-12T16:19:01.772Z","bundleUrl":"https://exp.host/@exponent/home/bundle","releaseChannel":"default","hostUri":"exp.host/@exponent/home"}
```

Comment:
I guess this will be updated once you rebase onto master?

Reply:
I already rebased onto master before I opened the PR. I had to change the bundleIdentifier to `dev.expo.Payments` to test on iOS so this is one of the changes we will have to revert before merging, yes.